link/src/main/java/com/stripe/android/link/LinkActivity.kt

```diff
@@ -119,7 +124,18 @@ internal class LinkActivity : ComponentActivity() {
 
                         LinkAppBar(
                             state = appBarState,
-                            onButtonClick = { viewModel.navigator.onBack() }
+                            onBackPress = viewModel::onBackPress,
+                            onLogout = viewModel::logout,
+                            showBottomSheetContent = {
+                                if (it == null) {
+                                    coroutineScope.launch {
+                                        sheetState.hide()
+                                        bottomSheetContent = null
+                                    }
+                                } else {
+                                    bottomSheetContent = it
+                                }
+                            }
```

Comment:
Same approach as with the other bottom sheet. I wish we could use the `BottomSheetNavigator` (from `accompanist-navigation-material`; basically adding the bottom sheets as a NavHost destination) for this, but that would require quite a substantial refactoring.